cmd/gameserverapi/main.go

```diff
@@ -144,6 +157,15 @@ func listGameServeBuilds(c *gin.Context) {
 	}
 }
 
+// @Summary get a game server build by buildName and namespace
```

Comment:
can we call them "GameServer" and "GameServerBuild" to be consistent with the rest of the documentation? Thank you!

Reply:
sure, will do!

Reply:
@dgkanatsios - Could you help with this error - "go utils need to be executed", which go-utils to use?

Reply:
this is the step in the pipeline that shows which utilities are executed. https://github.com/PlayFab/thundernetes/blob/main/.github/workflows/unit-tests.yml#L29

Reply:
Right, but how do I go about fixing it?